projectile-rails.el

```diff
@@ -204,6 +204,20 @@
 (defvar-local projectile-rails-zeus-sock nil
   "The path to the Zeus socket file")
 
+(defvar projectile-rails-generators
+  '(("assets" (("app/assets/javascripts/" "app/assets/javascripts/\\(.+\\)\\.coffee$")))
```

Comment:
You are right, it should be changed to: 
`(("app/assets/" "app/assets/\\(?:stylesheets\\|javascripts\\)/\\(.+?\\)\\..+$))`
I will modify that.